extras/paid-books.md

```diff
@@ -91,6 +91,11 @@ Name | Author | ISBN
 ### Cloud Computing
 
 ### Math (Linear Algebra)
+Name | Author | ISBN
+:-- | :--: | :--:
+[Linear Algebra Done Right third edition](http://www.amazon.in/Linear-Algebra-Right-Undergraduate-Mathematics/dp/0387982582) | Sheldon Axler | 0387982582
```

Comment:
To keep in the same format as the other links, I'd use the amazon.com link (and maybe capitalize Third Edition?)